apps/financial-aid/web-osk/src/utils/hooks/useApplication.ts

```diff
@@ -41,6 +41,7 @@ const useApplication = () => {
         const files = formatFiles(form.taxReturnFiles, FileType.TAXRETURN)
           .concat(formatFiles(form.incomeFiles, FileType.INCOME))
           .concat(formatFiles(form.otherFiles, FileType.OTHER))
+          .concat(formatFiles(form.taxReturnFromRskFile, FileType.TAXRETURN))
```

Comment:
Do we want to differ from the tax that we get from user vs rsk?

Reply:
We have too, if we don't then the user will get to the tax form screen and that will look like he has already uploaded a file. And that screen has changes depending if this file was successfully fetched or not.

Reply:
talk IRL